.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
Is it possible you add all the .vs and debug log into .gitignore. 
Unless there's a reason that the config for vscode needs to be shared with everyone, it shouldnt be committed.